ihatemoney/forms.py

```diff
@@ -216,6 +218,15 @@ def save(self):
             logging_preference=self.logging_preference,
             default_currency=self.default_currency.data,
         )
+        # Check that the member name is not empty so you don't insert an empty
+        # Person object if no member name is given
+        if self.member_name.data != "":
+            # Create a person object and link it to this project
+            member = Person(project_id=self.id.data, name=self.member_name.data)
+            # Add the new Person object to the database
+            db.session.add(member)
+            # Commit the database changes
```

Comment:
I believe the code is self-explanatory here, we could remove the last two comments I believe.